Backend/src/test/java/com/onbank/ftp/FtpClientTest.java

```diff
@@ -0,0 +1,76 @@
+package com.onbank.ftp;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockftpserver.fake.FakeFtpServer;
+import org.mockftpserver.fake.UserAccount;
+import org.mockftpserver.fake.filesystem.DirectoryEntry;
+import org.mockftpserver.fake.filesystem.FileEntry;
+import org.mockftpserver.fake.filesystem.FileSystem;
+import org.mockftpserver.fake.filesystem.UnixFakeFileSystem;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class FtpClientTest {
+
+    private FakeFtpServer fakeFtpServer;
+    @Value("${onbank.server}")
```

Comment:
Jeśli zrobimy component z naszego klienta to nie ma potrzeby deklaracji tych zmiennych w tym miejscu, zostaną one pobrane z propertisów testówych